bofire/strategies/predictives/sobo.py

```diff
@@ -240,13 +244,46 @@ def _get_objective_and_constraints(
                 objective=get_multiplicative_botorch_objective(  # type: ignore
                     outputs=self.domain.outputs,
                     experiments=self.experiments,
+                    adapt_weights_to_1_inf=True,
                 ),
             ),
             None,
             1e-3,
         )
 
 
+class MultiplicativeAdditiveSoboStrategy(SoboStrategy):
+    def __init__(
+        self,
+        data_model: MultiplicativeAdditiveDataModel,
+        **kwargs,
+    ):
+        self.additive_features = data_model.additive_features
+        super().__init__(data_model=data_model, **kwargs)
+
+    def _get_objective_and_constraints(
+        self,
+    ) -> Tuple[
+        GenericMCObjective,
+        Union[List[Callable[[torch.Tensor], torch.Tensor]], None],
+        Union[List, float],
+    ]:
+        # we absorb all constraints into the objective
+        assert self.experiments is not None, "No experiments available."
+        return (
+            GenericMCObjective(
```

Comment:
How do you prevent if you want to maximize a negative value in combination with a second feature on which some kind of desirability function between 0 and 1 is placed, that the optimizer just drives the desirability function on the second objective to 0. 

This would technically be also the same for minimizing a positive value, since minimization is achived by multiplying with -1.